src/lamp_py/runtime_utils/remote_files.py

```diff
@@ -8,98 +20,115 @@ class S3Location:
     wrapper for a bucket name and prefix pair used to define an s3 location
     """
 
-    bucket_name: str
-    file_prefix: str
+    bucket: str
+    prefix: str
 
-    def get_s3_path(self) -> str:
-        """generate the full s3 url that can be used to describe the location"""
-        return f"s3://{self.bucket_name}/{self.file_prefix}"
+    @property
+    def s3_uri(self) -> str:
+        """generate the full s3 uri for the location"""
+        return f"s3://{self.bucket}/{self.prefix}"
 
 
-class RemoteFileLocations:
-    """
-    Constant S3 File Locations Used in Bus Performance Manager
-    """
+# files ingested from delta
+rt_vehicle_positions = S3Location(
+    bucket=S3_SPRINGBOARD,
+    prefix=os.path.join(LAMP, "RT_VEHICLE_POSITIONS"),
+)
```

Comment:
Pulled all of these declarations out of the class to reduce the verbosity of referencing them in other parts of the application.

Reply:
makes sense. it was too long and too gross already.